kustomization-template.yml

```diff
@@ -34,13 +34,66 @@ configMapGenerator:
       - grpcendpoint=__GRPCENDPOINT__
       - maxgrpcpayload=__MAXGRPCPAYLOAD__
       - uuid=__UUID__
-patchesJson6902:
+# patchesJson6902:
 #   - target:
 #       group: "apps"
 #       version: v1
 #       kind: DaemonSet
 #       name: kappa-agent
 #     path: kappa-bytecode.yml
```

Comment:
:scissors: remove these commented lines maybe?

Reply:
Yeah, I need to dig more into the various bytecode requirements .. do we even need custom bytecodes anymore, and the cleanest way to manage them if we do.

This section has been (mostly) commented out for almost two years.  I'm leaving it in (for now) as a reminder to me whenever I look at this to poke a stick at the bytecode requirements when I get some time and if we need them, should we do more/less with this kustomization patch.

But, yeah, I'd like to get rid of it soon-ish.